From: Christian Marillat <marillat@deb-multimedia.org>
Date: Thu, 6 Jul 2017 15:59:17 +0200
Subject: Google-test from DEB repository

Use google-test provided by Debian distribution
instead of the version attached as git submodule
---
 Tests/Tests.pro | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/Tests/Tests.pro b/Tests/Tests.pro
index ed8d618..1ae97dc 100644
--- a/Tests/Tests.pro
+++ b/Tests/Tests.pro
@@ -30,10 +30,10 @@ CONFIG += openmvg-flags glad-flags
 
 !noexpat: CONFIG += expat
 
-INCLUDEPATH += google-test/include
-INCLUDEPATH += google-test
-INCLUDEPATH += google-mock/include
-INCLUDEPATH += google-mock
+INCLUDEPATH += /usr/src/googletest/googletest/include
+INCLUDEPATH += /usr/src/googletest/googletest
+INCLUDEPATH += /usr/src/googletest/googlemock/include
+INCLUDEPATH += /usr/src/googletest/googlemock
 
 
 QMAKE_CLEAN += ofxTestLog.txt test_dot_generator0.jpg
@@ -41,8 +41,8 @@ QMAKE_CLEAN += ofxTestLog.txt test_dot_generator0.jpg
 include(../global.pri)
 
 SOURCES += \
-    google-test/src/gtest-all.cc \
-    google-mock/src/gmock-all.cc \
+    /usr/src/googletest/googletest/src/gtest-all.cc \
+    /usr/src/googletest/googlemock/src/gmock-all.cc \
     BaseTest.cpp \
     Hash64_Test.cpp \
     Image_Test.cpp \
